plugins/env.go

```diff
@@ -0,0 +1,31 @@
+package plugins
+
+import (
+	"os"
+
+	"github.com/hashicorp/go-envparse"
+)
+
+// ParseEnvFile returns a slice of key/value pairs parsed from the file at filepath.
+// As a special case, empty filepath returns nil without error.
+func ParseEnvFile(filepath string) ([]string, error) {
+	if filepath == "" {
+		return nil, nil
+	}
+	f, err := os.Open(filepath)
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		_ = f.Close()
+	}()
+	m, err := envparse.Parse(f)
```

Comment:
I updated the changelog. Does it need to say more?

Reply:
> we have gone from a limited, specific set of env var to an unlimited set.

The env was already controlled by the node op since the host env is inherited. This just gives them fine gained control, per-plugin.

> it seems like you might be able to use this as a hook into parts of the system that shouldn't be hooked into (like the db?) maybe nbd since whoever is running the software has full control over the env anyway.

I don't fully grok the DB hook, but yeah that would already be possible before this change.